src/main/java/sirius/web/services/ServiceCall.java

```diff
@@ -176,7 +153,9 @@ protected void invoke(StructuredService serv) {
             // If the user unexpectedly closes the connection, we do not need to log an error...
             Exceptions.ignore(ex);
         } catch (Exception t) {
-            handle(null, t);
+            if (!handle(t)) {
```

Comment:
invoke handle just to log the exception and add the if (ctx.isResponseCommitted()) { check here